usefulUtilities/tabStatus/appResources/appData/resources/js/tabStatus_ui.js

```diff
@@ -0,0 +1,606 @@
+    See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+*/
+
+
+// *************************************
+// START UTILITY
+// *************************************
+// #region UTILITY
+
+
+// logging function for the browser
+function l(label, data, i){
+    data = JSON.stringify(data) + " " || "";
+    if (typeof(i) === "number"){
+        i = i;
+    } else {
+        i = "";
+    }
+    console.log("\n" + label + ": " + data + i +"\n");
+}
+
+
+// turn the first letter of each word into uppercase
+function makeUpperCase(text){
+    text = text.toLowerCase()
+        .split(' ')
+        .map((word) => word.charAt(0).toUpperCase() + word.substring(1))
+        .join(' ');
+    return text;
+}
+
+
+// replace the api call with the new information
+let statusURL = "https://highfidelity.co/api/statusIndicator?type=heartbeat&username=<username>&displayName=<display name>&status=<status, 150 char max>&teamName=<team name>"
+let replaceUserName = "<username>";
+let replaceDisplayName = "<display name>";
+let replaceStatus = "<status, 150 char max>";
+let replaceTeamName = "<team name>";
+function replaceURL(status, teamName){
+    statusURL = statusURL.replace(replaceUserName, username)
+    statusURL = statusURL.replace(replaceDisplayName, displayName)
+    statusURL = statusURL.replace(replaceStatus, status)
+    statusURL = statusURL.replace(replaceTeamName, teamName)
+    return statusURL;
+}
+
+
+// Helper function to see which index has a certain phrase
+function hasInArray(array, string){
+    for (i = 0; i < array.length; i++){
+        if (array[i].indexOf(string) > -1) {
+            return i;
+        }
+    }
+    return -1
+}
+
+
+// Gets items in an array of objects and turns them into an array of strings
+function objectToStringArrayMap(list){
+    let stringList = list.map(item => {
+        let fullString = "";
+        for (let key in item){
+            fullString += " " + String(item[key]).toLowerCase();
+        }
+        return fullString;
+    })
+    return stringList
+}
+
+
+// #endregion
+// *************************************
+// END UTILITY
+// *************************************
+
+// *************************************
+// START handlers
+// *************************************
+// #region handlers
+
+
+// handle the user submiting a status update
+function handleFormSubmit() {
+    let statusForm = document.getElementById('statusForm');
+    let status = document.getElementById('status');
+    let input = document.getElementById('filter_members');
+
+    let formData = new FormData(statusForm);
+    var formObject = {};
+    for (var pair of formData.entries()) {
+        formObject[pair[0]] = pair[1];
+    }
+    
+    let url = replaceURL(formObject.status, formObject.teamname);
+    fetch(url)
+        .catch(error => {
+            console.log(error)
+        });
+
+
+    showList();
+}
+
+
+// handle the user submitting a go back request
+function handleGoBack(){

```

Comment:
why do you need this wrapper function if all it's doing is calling one other function?